internal/driver/glfw/window_desktop.go

```diff
@@ -314,7 +314,10 @@ func (w *window) detectScale() float32 {
 		return 1.0
 	}
 
-	widthMm, _ := monitor.GetPhysicalSize()
+	widthMm, heightMm := monitor.GetPhysicalSize()
+	if widthMm == 60 && heightMm == 60 { // some sort of failure - mostly on Steam Deck
+		return 1
```

Comment:
Nit: I think the `return 1` statements should look all the same. Either with or without `.0`. Personally, I would leave it out.